### PR TITLE
Feature/53155 - heading

### PR DESCRIPTION
**Pull request by talalus**
- update classes from "heading" to "a-heading" 
- add unit test
- add knobs

### CODE REVIEW

File: src/atoms/heading/Heading.spec.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { mount } from '@vue/test-utils'
+import AHeading from './Heading.vue'
+
+describe('Heading', () => {
+  it('has default structure', () => {
+    const wrapper = mount(AHeading, {
+      propsData: {
+        level: 1,
+        tag: 'span'
```
Comment (szafran89) [inline src/atoms/heading/Heading.spec.js]: tag is not part of the default structure

File: src/atoms/heading/Heading.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          <span>Heading level 1</span>
+        `
+      }
+    })
+
+    expect(wrapper.is('h1')).toBe(true)
+    expect(wrapper.find('.a-heading span').exists()).toBe(true)
+    expect(wrapper.find('.a-heading span').text()).toEqual('Heading level 1')
+  })
+})
```
Comment (szafran89) [inline src/atoms/heading/Heading.spec.js]: and more tests related to `tag` prop

File: src/atoms/heading/Heading.stories.js [hunk 3]
```diff
@@ -1,27 +1,64 @@
...
+`
+
+const levels = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6
+]
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: add knobs for tab property

File: src/atoms/heading/Heading.stories.js [hunk 4]
```diff
@@ -1,27 +1,64 @@
...
+  ---
+`
+
+const levels = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: you can keep it in one line

File: src/atoms/heading/Heading.stories.js [hunk 5]
```diff
@@ -1,27 +1,64 @@
...
+]
 
 storiesOf('Atoms/Heading', module)
-  .addParameters({ info: true })
+  .addParameters({ info })
   // @vue/component
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: remove these comments from stories files

File: src/atoms/heading/Heading.stories.js [hunk 6]
```diff
@@ -1,27 +1,64 @@
...
   .add('Default', () => ({
     components: { AHeading },
+    props: {
+      levelKnobs: {
+        default: select('Heading level', levels, 1),
+        type: Number
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: remove type

File: src/atoms/heading/Heading.spec.js (hunk 2 above)
Comment (talalus) [reply src/atoms/heading/Heading.spec.js]: done

**Comment by szafran89** [comment]
CI issue https://circleci.com/gh/SnowdogApps/alpaca-ui/1498?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: src/atoms/heading/Heading.stories.js [hunk 7]
```diff
@@ -1,27 +1,60 @@
...
+      >
+        {{ textKnobs }}
       </a-heading>
     `
   }))
-  // @vue/component
   .add('Custom tag', () => ({
     components: { AHeading },
+    props: {
+      levelKnobs: {
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: We can remove `Custom tag` story b/c we provided knobs

File: src/atoms/heading/Heading.stories.js [hunk 8]
```diff
@@ -1,27 +1,60 @@
 import { storiesOf } from '@storybook/vue'
 
 import AHeading from './Heading.vue'
+import { select, text } from '@storybook/addon-knobs'
+
+const info = `
+  ---
+  Check **Knobs** tab to edit component properties dynamically. All HTML six levels of headings are available.
+
```
Comment (szafran89) [inline src/atoms/heading/Heading.stories.js]: Please remove empty line

**Pull request merged**